views/pages/uploadImage.ejs

```diff
@@ -1,48 +1,56 @@
-            <span class="letters">
-                Meddy.
-            </span>
-            <div class="scan-line"></div>
-        </div>
-    </div>
-    <div class="upload-container">
-        <div id="image-place" class="place-holder">
-            <div class="imageplaceholder">
+    <div class="container">
+        <div class="upload-container">
+            <div id="image-place" class="place-holder">
+                <div class="imageplaceholder">
 
-                <% if(typeof image == 'string' && image){ %>
-                <img src="/image/<%= image %>" alt="" class="show-image">
-                <% }else{ %>
-                <img src="assets/svg/imageload.svg" alt="">
-                <% } %>
+                    <% if(typeof image == 'string' && image){ %>
+                    <img src="/image/<%= image %>" alt="" class="show-image">
+                    <% }else{ %>
+                    <img src="assets/svg/imageload.svg" alt="">
+                    <% } %>
 
-                <article>
-                    <p>
-                        gebruik je camera om een foto te maken
+                    <article>
+                        <p>
+                            gebruik je camera om een foto te maken
 
-                    </p>
-                    <span> of </span>
-                </article>
-            </div>
-            <div class="form-container">
-                <form action="/upload-image-page" method="POST" enctype="multipart/form-data">
-                    <input type="file" name="scanmeds" id="file" class="inputfile" />
-                    <label for="file">kies een foto</label>
-                    <button type="submit" class="upload">scan de foto</button>
-                </form>
+                        </p>
+                        <span> of </span>
+                    </article>
+                </div>
+                <div class="form-container">
+                    <form action="/upload-image-page" method="POST" enctype="multipart/form-data">
+                        <input type="file" name="scanmeds" id="file" class="inputfile" />
+                        <label for="file">kies een foto</label>
+                        <button type="submit" class="upload">scan de foto</button>
+                    </form>
+                </div>
             </div>
         </div>
-    </div>
+        <section class="meds-result">
+            <div class="loading-state">
+                <div>
+                    <span class="letters">
+                        Meddy.
+                    </span>
+                    <div class="scan-line"></div>
+                </div>
+            </div>
+            <% if( typeof meds == 'object' && !meds.rating && !meds.activeIngredient == ''){%>
+            <p><%= text  %> </p>
+            <%- include('../partials/scanner_results') %>
+
+            <% } %>
+            <% if(typeof text == 'string' && text.length <= 3 || typeof meds == 'object' && meds.rating < 0.3 ){ %>
```

Comment:
Isn't this similar to the if statement in the uploadpage? Correct me if I'm wrong 🙈

Reply:
it's the same

Reply:
Is there a particularly reason for doing so?

Reply:
I was thinking of giving different error statement via the client by creating this kind of if statement.

Reply:
ah